src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -290,7 +324,11 @@ export class MgtPersonCard extends MgtTemplatedComponent {
 
     // Phone
     let phone: TemplateResult;
-    if (userPerson.businessPhones && userPerson.businessPhones.length > 0) {
+    if (
+      (userPerson.businessPhones && userPerson.businessPhones.length > 0) ||
+      (personPerson.phones && personPerson.phones.length > 0) ||
+      (contactPerson.businessPhones && contactPerson.businessPhones.length > 0)
+    ) {
```

Comment:
Same here. Can be simplified:

```suggestion
    if (
      (userPerson.businessPhones && userPerson.businessPhones.length > 0) ||
      (personPerson.phones && personPerson.phones.length > 0))
    ) {
```